### ISSUE TITLE
How to use custom loss fuction?

### ISSUE THREAD

**Issue by bughunter99** [fchollet/keras#6334]
Hi, I want to use custom loss function, not like by calculating from difference between y_true, y_pred.
For examples, I mean another gap something like y_pred and another model's result y_pred2.
I have found a week, but I couldn't find similar one.
Is it possible this in keras?
Thanks.

**Comment by ParthaEth** [comment]
you can take the outputs of both the layers in a lambda layer perform a point-wise subtraction and put this out as final output of your model. Now you can provide targets to be zeros. This should do exactly what you want.

**Comment by bughunter99** [comment]
Thanks for reply. What I want to do is to solve following link : https://github.com/fchollet/keras/issues/6236 . Do you have any idea for this? I haven't seen related source for this.

**Comment by bughunter99** [comment]
This had been solved like : https://stackoverflow.com/questions/43368241/student-teacher-model-in-keras
